jekyll/_cci2/project-walkthrough.md

```diff
@@ -242,11 +242,11 @@ Notes on the added keys:
 
 - Each command runs in a new shell, so the virtual environment that was activated in the dependencies installation step is activated again in this final `run:` key with `. venv/bin/activate`. 
 - The `store_artifacts` step is a special step. The `path:` is a directory relative to the project’s `root` directory where the files are stored. The `destination:` specifies a prefix chosen to be unique in the event that another step in the job produces artifacts in a directory with the same name. CircleCI collects and uploads the artifacts to S3 for storage.
-- When the {% comment %} TODO: Job {% endcomment %}build completes, artifacts appear in the CircleCI Artifacts tab:
+- When a job completes, artifacts appear in the CircleCI Artifacts tab:
```

Comment:
The build of a job completes? A job completes running? (This comment just echoes my own earlier confusion, so it may get dismissed with all the above.)

Reply:
My understanding is that the artifacts for a particular job will appear after the associated job completes. Unless, **all** artifacts for **all** jobs have to wait for the  the **entire** build to finish. @ndintenfass?

Reply:
No, artifacts are job-specific.